src/core/common/code_utils.hpp

```diff
@@ -37,7 +37,6 @@
 #include "openthread-core-config.h"
 
 #include "utils/static_assert.hpp"
-#include "utils/wrap_stdbool.h"
```

Comment:
Any reason for removing `stdbool.h`? 

I think we need it for definition of `false` (and `true`) which is used in the macros `VerifyOrExit`, 'SuccessOrExit`

Reply:
Have tried locally. Will add it back